README.md

```diff
@@ -21,14 +21,20 @@ Learn more:
 ### Downloading
 You can download ZAP enabled with the HUD from any of: 
```

Comment:
Maybe replace `download` with `use`?

Reply:
Changed to "try out" as per irc conversation